instrumentation/google.golang.org/grpc/otelgrpc/config.go

```diff
@@ -131,3 +134,37 @@ func (o meterProviderOption) apply(c *config) {
 func WithMeterProvider(mp metric.MeterProvider) Option {
 	return meterProviderOption{mp: mp}
 }
+
+type event int
```

Comment:
I am wondering why `event` is unexported and why we are not using the value a [bitmask](https://yourbasic.org/golang/bitmask-flag-set-clear/) approach. The same comment applies to the `otelhttp` package.

However, I think that in scope of this PR is it better to keep the consistency with `otelhttp` package.

Reply:
I created https://github.com/open-telemetry/opentelemetry-go-contrib/issues/3978